<ui version="4.0" >
 <class>AddContactDialog</class>
 <widget class="QWidget" name="AddContactDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Add Contact</string>
  </property>
  <property name="autoFillBackground" >
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="Contact_Tab" >
      <attribute name="title" >
       <string>Contact</string>
      </attribute>
      <widget class="QLabel" name="label" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>20</y>
         <width>189</width>
         <height>31</height>
        </rect>
       </property>
       <property name="palette" >
        <palette>
         <active>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>153</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>153</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>153</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>153</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>153</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>153</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font" >
        <font>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet" >
        <string>QLabel { color: #FF9900; font: 30px }</string>
       </property>
       <property name="text" >
        <string>Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nameLineEdit" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>50</y>
         <width>441</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font" >
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_2" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>110</y>
         <width>189</width>
         <height>31</height>
        </rect>
       </property>
       <property name="palette" >
        <palette>
         <active>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>153</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>153</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>153</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>153</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>153</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>153</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font" >
        <font>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet" >
        <string>QLabel { color: #FF9900; font: 30px }</string>
       </property>
       <property name="text" >
        <string>Number</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="numberLineEdit" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>140</y>
         <width>441</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font" >
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Email_Tab" >
      <attribute name="title" >
       <string>Email</string>
      </attribute>
      <widget class="QLineEdit" name="emailLineEdit" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>50</y>
         <width>441</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font" >
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_3" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>20</y>
         <width>189</width>
         <height>31</height>
        </rect>
       </property>
       <property name="palette" >
        <palette>
         <active>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>153</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>153</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>153</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>153</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>153</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>153</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font" >
        <font>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet" >
        <string>QLabel { color: #FF9900; font: 30px }</string>
       </property>
       <property name="text" >
        <string>Email</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Picture_Tab" >
      <attribute name="title" >
       <string>Picture</string>
      </attribute>
      <widget class="QLabel" name="pictureLabel" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>30</y>
         <width>251</width>
         <height>301</height>
        </rect>
       </property>
       <property name="text" >
        <string>No Picture</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QToolButton" name="browsePushButton" >
       <property name="geometry" >
        <rect>
         <x>280</x>
         <y>119</y>
         <width>171</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text" >
        <string>Browse...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="removePushButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="geometry" >
        <rect>
         <x>280</x>
         <y>190</y>
         <width>171</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text" >
        <string>Remove</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Ring_Tab" >
      <attribute name="title" >
       <string>Ring</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="savePushButton" >
       <property name="text" >
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
